CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Simplify the modules code ([PR #2237](https://github.com/alphagov/govuk_publishing_components/pull/2237))
```

Comment:
A changelog entry isn't necessary as this is an internal change: https://github.com/alphagov/govuk_publishing_components/blob/master/CHANGELOG.md#changelog